### PR TITLE
Add sdk demand for release pipeline

### PR DESCRIPTION
**Pull request by MenelvagorMilsom**

### CODE REVIEW

**Comment by MenelvagorMilsom** [comment]
@wiwei, should we add this for all pipelines?

File: pipelines/ci-release.yml [hunk 1]
```diff
@@ -12,6 +12,7 @@ jobs:
     demands:
     - Unity2018.3.7f1  # variable expansion not allowed here
     - COG-UnityCache-WUS2-01
+    - SDK_18352 -equals TRUE
```
Comment (wiwei) [inline pipelines/ci-release.yml]:
```suggestion
    - SDK_18362 -equals TRUE
```

**Comment by wiwei** [comment]
@MenelvagorMilsom I think that would be best getting this demand added for all pipelines.

**Comment by MenelvagorMilsom** [comment]
@keveleigh yeah, it does that sometimes

**Comment by macborow** [comment]
FYI, I got that "file not found" error when running a cloned mrtk_CI job against my fork. I thought it was due to an expired authorization token as re-authenticating seems to have fixed it. Perhaps it's an intermittent failure though as I think this build recovered after being forced again... I'll mention that to Nick and perhaps he can pass this info on to the ADO team.

**Pull request merged**